community/detectors/jira_cve_2022_0540/README.md

```diff
@@ -0,0 +1,16 @@
+# Jira Authentication Bypass Vulnerability Detector
```

Comment:
Please add more details about the 2 plugins here. 

Also remove trailing whitespaces at the end of each line, it's generating a linter error.